Source/setmaps.cpp

```diff
@@ -94,25 +94,30 @@ ObjectStruct &ObjectAtPosition(Point position)
 
 void AddSKingObjs()
 {
-	ObjectAtPosition({ 64, 34 }).SetMapRange({ 20, 7 }, { 23, 10 }, 1);
-	ObjectAtPosition({ 64, 59 }).SetMapRange({ 20, 14 }, { 21, 16 }, 2);
-	ObjectAtPosition({ 27, 37 }).SetMapRange({ 8, 1 }, { 15, 11 }, 3);
-	ObjectAtPosition({ 46, 35 }).SetMapRange({ 8, 1 }, { 15, 11 }, 3);
-	ObjectAtPosition({ 49, 53 }).SetMapRange({ 8, 1 }, { 15, 11 }, 3);
-	ObjectAtPosition({ 27, 53 }).SetMapRange({ 8, 1 }, { 15, 11 }, 3);
+	constexpr Rectangle region1 { { 20, 7 }, { 3, 3 } };
+	ObjectAtPosition({ 64, 34 }).InitializeLoadedObject(region1, 1);
+
+	constexpr Rectangle region2 { { 20, 14 }, { 1, 2 } };
+	ObjectAtPosition({ 64, 59 }).InitializeLoadedObject(region2, 2);
+
+	constexpr Rectangle region3 { { 8, 1 }, { 7, 10 } };

```

Comment:
would be nice if we could name them `smallSecretRoom`, `gate` and `largeSecretRoom`

Reply:
Thanks heaps, I hadn't got around to working out what these actually were. I'll update the PR now.